notebooks/COS/DayNight/DayNight.ipynb

```diff
@@ -0,0 +1,621 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# NOW, we make an association file for the unfiltered data files\n",
+    "\n",
+    "for f in file_locations_a:\n",
+    "    # Adding the file details to the association table\n",
+    "    rootnames = [f]  # MEMNAME\n",
+    "    types = [type_dict[fits.getval(f, 'EXPTYPE')]] # MEMTYPE\n",
+    "    included = [True] # MEMPRSNT\n",
+    "    \n",
+    "    # Adding the ASN details to the end of the association table\n",
+    "    # the rootname needs to be the full name, not just the rootname\n",
+    "    asn_root = os.path.basename(f.split('corrtag')[0][:-1]) # removing the extra \"_\" at the end of this\n",
+    "    rootnames.append(asn_root.upper()) # MEMNAME\n",
+    "    types.append('PROD-FP') # MEMTYPE\n",
+    "    included.append(True) # MEMPRSNT\n",
+    "    \n",
+    "    # Putting together the fits table\n",
+    "    c1 = fits.Column(name='MEMNAME', array=np.array(rootnames), format='40A') \n",
+    "    c2 = fits.Column(name='MEMTYPE', array=np.array(types), format='14A')\n",
+    "    c3 = fits.Column(name='MEMPRSNT', format='L', array=included)\n",
+    "    t = fits.BinTableHDU.from_columns([c1, c2, c3])\n",
+    "    \n",
+    "    # Writing the fits table\n",
+    "    t.writeto(asn_root.lower()+'_asn.fits', overwrite=True)\n",
+    "\n",
+    "    print('Saved: {}'.format(asn_root.lower()+'_asn.fits' ), \"in the cwd directory\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Again, we can see the contents of this association file below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Table.read(f\"{filename_root}_asn.fits\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id = calcosF></a>\n",
+    "## 1.3. Running the `calcos` pipeline\n",
+    "\n",
+    "Now we need to actually reduce the data using the `calcos` pipeline. If you have not already checked out our tutorial on [Running the `calcos` pipeline](), it contains vital information and is *highly recommended*.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "<font size=\"4 \">[If you haven't downloaded the files as in Section 3 of \"Setup.ipynb\", click to skip this cell!](#skipcellF)\n",
+    "\n",
+    "<font size=\"4 \">If you ran [Section 3 of our notebook on \"Setting up your environment\"](https://github.com/spacetelescope/notebooks/blob/master/notebooks/COS/Setup/Setup.ipynb), you likely do not have to download more reference files. \n",
```

Comment:
for me (chrome + notebook) this font size for some reason seems to stick for the next several cells... :zany_face: